advance-projects/leader282-Musico/index.js

```diff
@@ -0,0 +1,80 @@
+// getting all required elements
+const searchInput = document.querySelector(".searchInput");
+const input = searchInput.querySelector("input");
+const resultBox = searchInput.querySelector(".resultBox");
+const icon = searchInput.querySelector(".icon");
+let linkTag = searchInput.querySelector("a");
+let webLink;
+
+// if user press any key and release
+input.onkeyup = (e)=>{
+    let userData = e.target.value; //user enetered data
+    let emptyArray = [];
+    if(userData){
```

Comment:
You can use Guard Clause here to avoid the code nesting "if (!userData) return"

Reply:
Yeah, I found another way and implemented that